packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue

```diff
@@ -142,7 +144,7 @@ const knobs = ref<InterventionPolicyGroupForm>({
 	...props.config
 });
 
-const isNotEditable = computed(() => staticInterventions.value.length !== 1);
+// const isNotEditable = computed(() => staticInterventions.value.length !== 1);
```

Comment:
do we still need this? if so, put a todo. if not remove it

Reply:
will remove!